### ISSUE TITLE
Deduplicate inbound and outbound networks

### ISSUE THREAD

**Issue by zerebubuth** [tilezen/vector-datasource#1204]
* **What did you see?**

```
'all_bus_networks': [None, None],
'all_bus_shield_texts': ['3', '3'],
```

A road is quite often part of two bus routes, one for "inbound" and the other "outbound" in reverse. For the purposes of cartography, these are not really two separate routes.

* **What did you expect to see?**

Where the road is part of both inbound and outbound routes, they should be collapsed to a single entry in the `all_*_networks` and `all_*_shield_texts` lists.

@nvkelso, do we want to distinguish the de-duplicated value from the originals in some way? So that it's possible to tell the difference between a road that the bus goes along in both directions from a road that only has busses going one way or the other.

* **What map location are you having problems with?** _City and country are helpful, as well as tile coordinates or latitude / longitude._  

E.g: [Jackson St. SF](http://www.openstreetmap.org/way/225516711), part of trolley-bus route 3 ([inbound](http://www.openstreetmap.org/relation/2980504), [outbound](http://www.openstreetmap.org/relation/2980505)).

**Comment by nvkelso** [comment]
**Related:** I sometimes see inbound and outbound route shapes in the `transit` layer. That's a similar problem, but if we want to tackle that should be done in a separate ticket.

**Comment by zerebubuth** [comment]
> if we want to tackle [de-duplicating inbound and outbound route shapes] should be done in a separate ticket.

Yup, definitely! De-duping shapes is way harder than just de-duping the network name and shield text. We'd be looking at something like skeletron for the shape stuff.

**Comment by nvkelso** [comment]
Skeletron, ftw :)
